### DISCUSSION TITLE
Allow conditional user fields to operate in the same way as conditional entry fields

### DISCUSSION THREAD

**Discussion by SETU-WEB** [craftcms/cms#13481] [Ideas] [answered]
Currently you can add any field you want to the user fields section, and you can apply conditionals like you can on any entry field. But the conditionals don't operate in real time and the user doesn't see the conditional field until they hit save. Another downside of this is the save brings you back to the first tab and so the user doesn't see the change unless they click back into to the user fields tab. 

Its clunky and not user friendly for the user to keep saving for conditional fields to appear. I presume this is because auto save isn't enabled like on entry fields so the conditionals don't work in real time.

I use the user fields for staff profiles in university, and we have around 50 fields that the user can fill out. Not all fields are relevant to all users so having conditional fields operate in real time would really improve the process of creating a staff profile.

**Comment by brandonkelly** [comment]
Yeah agree we should do this, but switching the Edit User page to the new unified element editor will be a little challenging, so not sure when we’ll be able to get to it.

**Comment by brandonkelly** [comment, marked as answer]
Live conditional field support has been added to user slideouts in Craft 4.6.0, and full-page edit forms as well in Craft 5.0.0-alpha.5 🎉  (https://github.com/craftcms/cms/pull/14115)

**Comment by SETU-WEB** [comment]
Great stuff, just tested it on the craft 5 alpha and it works nicely
